app/assets/javascripts/discourse-i18n/src/index.js

```diff
@@ -4,6 +4,8 @@ if (window.I18n) {
   );
 }
 
+import * as Cardinals from "make-plural/cardinals";
```

Comment:
This is already a dependency of `@messageformat/runtime`, so I guess it’s not a problem to use it here.